staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -52,6 +69,9 @@ func NewBackOff(initial, max time.Duration) *Backoff {
 		Clock:           clock.RealClock{},
 		defaultDuration: initial,
 		maxDuration:     max,
+		randomSeedSource: func() int64 {
+			return time.Now().UTC().UnixNano()
```

Comment:
nit: use `rand.Uint64()`; rand should already be seeded correctly, so using it means client-go won't be opinionated about how that is done.

Reply:
I'm concerned about this, rand.Uint64() is part of the global seed lock.  That's somewhat concerning and may have a more negative impact than anticipated.  Would need to be tested at scale.

Reply:
I doubt that'll be an issue, but if it is, we could construct an intermediate rand instance for generating these seeds.